### PR TITLE
added user instructions to docs

### PR DESCRIPTION
**Pull request by robertoostenveld**
there is no point in having this in email attachments

### CODE REVIEW

File: docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+For example, change the README file and commit it.
+
+Check `Settings > Webhooks` and see if the delivery of the payload was succesful.
+There should be a green tickmark.
+
+## 5. Check the results on mentat
+
+Login to a mentat machine of choice, for example `mentat005.dccn.nl`.
+
+Go to your qaas results folder `/home/tg/rutvdee/.qaas/5126d168-e3f1-4c7f-b228-a57fbaf007c4`.
```
Comment (aardkronkel) [inline docs/instructions.md]: Please replace `/home/tg/rutvdee/.qaas/5126d168-e3f1-4c7f-b228-a57fbaf007c4` with something like `/home/dccngroup/dccnuser/.qaas/5126d168-e3f1-4c7f-b228-a57fbaf007c4`

**Review by aardkronkel — COMMENTED** [review]
Good point.

**Comment by hurngchunlee** [comment]
Since the client side tool is integrated with the unified cluster tool command replacing the current cluster-xxx scripts, and the functionality is related to the HPC, I think the final complete instruction should be part of the HPC wiki.

**Comment by robertoostenveld** [comment]
I think that the HPC wiki should be merged (one way or another) with the intranet. Storage, project quota, installed software, archiving, etc., are also all part of the same research compute environment.

**Pull request merged**